.github/workflows/ts-rpc-test.yml

```diff
@@ -2,37 +2,25 @@ name: TypeScript RPC Client Test
 
 on:
   pull_request:
-    paths: ['packages/nitro-rpc-client', '.github/workflows/ts-rpc-test.yml']
```

Comment:
This might be a little controversial, since this action takes 4 or 5 minutes to run. It will also force us to fix the TS client before merging any breaking changes to the go-nitro.